### PR TITLE
Added NextJs to list of projects.

### PR DESCRIPTION
**Pull request by krishnagit2402**

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -514,6 +514,13 @@ const projectList = [
     githubLink: 'https://github.com/tensorflow/tensorflow',
     description: 'A Machine Learning library in Python for implementing Machine Learning and Deep Learning models',
     tags: ['python', 'c++', 'machine learning', 'math', 'deep learning', 'neural network']
+  },
+  {
+    name: 'Next',
+    imageSrc: 'https://camo.githubusercontent.com/1f8dec51cb01842d7bb7a7cd50ade17c75c5e3bd/68747470733a2f2f6173736574732e7a6569742e636f2f696d6167652f75706c6f61642f76313533383336313039312f7265706f7369746f726965732f6e6578742d6a732f6e6578742d6a732e706e67',
```
Comment (sara-02) [inline src/components/ProjectList/listOfProjects.js]: I am getting a `Bad Signature` error when trying to open this Image URL.

Comment (krishnagit2402) [reply src/components/ProjectList/listOfProjects.js]: I don't get any error but the background is being black. However, that is what i got from githubusercontent. Can i provide external link for logo?

Comment (sara-02) [reply src/components/ProjectList/listOfProjects.js]: Can you paste the image link here.

Comment (krishnagit2402) [reply src/components/ProjectList/listOfProjects.js]: https://cloud.githubusercontent.com/assets/13041/19686250/971bf7f8-9ac0-11e6-975c-188defd82df1.png

**Comment by sara-02** [comment]
@krishnagit2402 Thanks :tada:

**Pull request merged**